### ISSUE TITLE
[FEAT] Expense tracking using receipt and OCR

### ISSUE THREAD

**Issue by ybmirz** [flow-mn/flow#185]
### Description

Hey! 
Been waiting for someone to make an open source expense tracker, and glad to see I finally came across one. 
This feature has been on my mind for a bit, ever since I started using any expense tracker application - while most YNAB and alike rely on bank notification to track expenses, I was thinking a simple OCR model implemented in an on-device app for receipts would be a much simpler method of tracking expenses. 

I plan to make my own contribution to this feature, so keep a look out.
Let me know what you guys think :D

**Comment by sadespresso** [comment]
Definitely a good idea!

Tho I don't have much experience with OCRing, I will definitely look into it.

**Comment by sadespresso** [comment]
So I've been researching on this for a while. Generally, it'd require combination of OCR and another form of technology that'd make sense of the image (structure, semantics, etc) - basically a trained AI model.

Paid solutions cost around $0.08 per image, which is very expensive, and I'm definitely not gonna introduce paid plan to Flow.

**Comment by sadespresso** [comment]
Plus, on-device OCR (teserract) isn't that fast.

**Comment by sadespresso** [comment]
I don't think this matter will be worked on anytime soon unless there is practical, fast trained model, or another technology that'd allow us to get data from receipt images
